### PR TITLE
docs: config file - provide better link to Finder docs

### PR DESCRIPTION
**Pull request by 94noni**
ref https://github.com/PHP-CS-Fixer/PHP-CS-Fixer/discussions/6991

### CODE REVIEW

File: doc/config.rst [hunk 1]
```diff
@@ -14,14 +14,25 @@ With the ``--config`` option you can specify the path to the
...
+
 .. code-block:: php
 
     <?php
 
     $finder = PhpCsFixer\Finder::create()
-        ->exclude('somedir')
-        ->notPath('src/Symfony/Component/Translation/Tests/fixtures/resources.php')
         ->in(__DIR__)
+        ->exclude('somedir')
```
Comment (mvorisek) [inline doc/config.rst:34]: example with path/file would be helpful as well

Comment (94noni) [reply doc/config.rst:34]: indeed, lets add it

File: doc/config.rst [hunk 2]
```diff
@@ -14,14 +14,27 @@ With the ``--config`` option you can specify the path to the
...
+Consider your codebase as follow:
+
+```
+├── .php-cs-fixer.php
+└── src
+    ├── foo.php
+    ├── bar.php
+    ├── somedir
+    └── maindir
+        ├── submaindir
```
Comment (Wirone) [inline doc/config.rst:26]:
```suggestion
        ├── subdir
```

File: doc/config.rst [hunk 3]
```diff
@@ -14,14 +14,27 @@ With the ``--config`` option you can specify the path to the
...
+
 .. code-block:: php
 
     <?php
 
     $finder = PhpCsFixer\Finder::create()
-        ->exclude('somedir')
-        ->notPath('src/Symfony/Component/Translation/Tests/fixtures/resources.php')
-        ->in(__DIR__)
+        ->in(__DIR__) // 'src'
```
Comment (Wirone) [inline doc/config.rst:35]:
```suggestion
        ->in(__DIR__)
```

`__DIR__` is one level higher than `src` so this comment is wrong.

File: doc/config.rst [hunk 4]
```diff
@@ -14,14 +14,27 @@ With the ``--config`` option you can specify the path to the
...
+        ->exclude(['somedir', 'foo.php'])
```
Comment (Wirone) [inline doc/config.rst:36]:
```suggestion
        ->exclude('somedir')
```

As you can see below:

> **NOTE**: ``exclude`` will work only for directories, so if you need to exclude file, try ``notPath``.

**Review by Wirone — CHANGES_REQUESTED** [review]

File: doc/config.rst (hunk 1 above)
Comment (Wirone) [reply doc/config.rst]: [See](https://github.com/PHP-CS-Fixer/PHP-CS-Fixer/pull/6992#discussion_r1205425410).

Comment (mvorisek) [reply doc/config.rst]: I do not see it. I meant the oposite of `->notPath('bar.php')`, ie. to include a single file like `config.php` which is not inside included `__DIR__ . '/src'`.

**Comment by 94noni** [comment]
I will be afk for some days
Feel free to continue this PR as you desire to match your doc standards :)

File: doc/config.rst [hunk 5]
```diff
@@ -14,14 +14,27 @@ With the ``--config`` option you can specify the path to the
...
+├── .php-cs-fixer.php
+└── src
+    ├── foo.php
+    ├── bar.php
+    ├── somedir
+    └── maindir
+        ├── subdir
+        └── baz.php
+```
+
```
Comment (keradus) [inline doc/config.rst:29]: this doesn't work and tbh, I do not see big value of adding it
Screenshot 2023-10-29 at 20 51 52

```suggestion
```

File: doc/config.rst (hunk 3 above)
Comment (keradus) [inline doc/config.rst:35]:
```suggestion
        ->in(__DIR__)
```

File: doc/config.rst (hunk 4 above)
Comment (keradus) [inline doc/config.rst:36]:
```suggestion
        ->exclude(['dir_a', 'dir_b'])
```

File: doc/config.rst [hunk 6]
```diff
@@ -14,14 +14,27 @@ With the ``--config`` option you can specify the path to the
...
 
     <?php
 
     $finder = PhpCsFixer\Finder::create()
-        ->exclude('somedir')
-        ->notPath('src/Symfony/Component/Translation/Tests/fixtures/resources.php')
-        ->in(__DIR__)
+        ->in(__DIR__) // 'src'
+        ->exclude(['somedir', 'foo.php'])
+        ->notPath('bar.php')
```
Comment (keradus) [inline doc/config.rst:37]:
```suggestion
        ->notPath(['foo.php', 'dir_c/bar.php'])
```

File: doc/config.rst [hunk 7]
```diff
@@ -36,7 +36,7 @@ The example below will add two rules to the default list of PSR12 set rules:
 **NOTE**: ``exclude`` will work only for directories, so if you need to exclude file, try ``notPath``.
```
Comment (keradus) [inline doc/config.rst:36]:
```suggestion
**NOTE**: ``exclude`` will work only for directories, so if you need to exclude file, use ``notPath``.
```

File: doc/config.rst [hunk 8]
```diff
@@ -33,10 +33,10 @@ The example below will add two rules to the default list of PSR12 set rules:
         ->setFinder($finder)
     ;
 
-**NOTE**: ``exclude`` will work only for directories, so if you need to exclude file, try ``notPath``.
+**NOTE**: ``exclude`` will work only for directories, so if you need to exclude file, use ``notPath``.
 Both ``exclude`` and ``notPath`` methods accept only relative paths to the ones defined with the ``in`` method.
```
Comment (keradus) [inline doc/config.rst:37]:
```suggestion
Both ``exclude`` and ``notPath`` methods accept only relative paths to the ones defined with the ``in`` method, can be called multiple times and accept string or array of strings.
```

File: doc/config.rst [hunk 9]
```diff
@@ -19,9 +19,9 @@ The example below will add two rules to the default list of PSR12 set rules:
     <?php
 
     $finder = PhpCsFixer\Finder::create()
-        ->exclude('somedir')
-        ->notPath('src/Symfony/Component/Translation/Tests/fixtures/resources.php')
         ->in(__DIR__)
+        ->exclude(['dir_a', 'dir_b'])
+        ->notPath(['foo.php', 'dir_c/bar.php'])
```
Comment (keradus) [inline doc/config.rst:24]:
```suggestion
        ->exclude('somedir')
        ->>notPath('src/Symfony/Component/Translation/Tests/fixtures/resources.php')
        ->in(__DIR__)
```

File: doc/config.rst [hunk 10]
```diff
@@ -33,10 +33,10 @@ The example below will add two rules to the default list of PSR12 set rules:
         ->setFinder($finder)
     ;
 
-**NOTE**: ``exclude`` will work only for directories, so if you need to exclude file, try ``notPath``.
-Both ``exclude`` and ``notPath`` methods accept only relative paths to the ones defined with the ``in`` method.
+**NOTE**: ``exclude`` will work only for directories, so if you need to exclude file, use ``notPath``.
```
Comment (keradus) [inline doc/config.rst:36]:
```suggestion
**NOTE**: ``exclude`` will work only for directories, so if you need to exclude file, try ``notPath``.
```

File: doc/config.rst [hunk 11]
```diff
@@ -33,10 +33,10 @@ The example below will add two rules to the default list of PSR12 set rules:
...
+Both ``exclude`` and ``notPath`` methods accept only relative paths to the ones defined with the ``in`` method, can be called multiple times and accept string or array of them.
```
Comment (keradus) [inline doc/config.rst:37]:
```suggestion
Both ``exclude`` and ``notPath`` methods accept only relative paths to the ones defined with the ``in`` method.
```

File: doc/config.rst [hunk 12]
```diff
@@ -20,7 +20,7 @@ The example below will add two rules to the default list of PSR12 set rules:
 
     $finder = PhpCsFixer\Finder::create()
         ->exclude('somedir')
-        ->notPath('src/Symfony/Component/Translation/Tests/fixtures/resources.php')
+        ->>notPath('src/Symfony/Component/Translation/Tests/fixtures/resources.php')
```
Comment (keradus) [inline doc/config.rst:23]:
```suggestion
        ->notPath('src/Symfony/Component/Translation/Tests/fixtures/resources.php')
```

**Comment by keradus** [comment]
thanks for your effort on this PR!

I decided to keep only the link, and redo the other part of docs https://github.com/PHP-CS-Fixer/PHP-CS-Fixer/pull/7396

**Pull request merged**